android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -612,6 +637,18 @@ public void setTargetingDataMaximumAdContentRating(final int maximumAdContentRat
         sdk.getTargetingData().setMaximumAdContentRating( getAppLovinAdContentRating( maximumAdContentRating ) );
     }
 
+    @ReactMethod
+    public void getTargetingDataMaximumAdContentRating(final Promise promise)
+    {
+        if ( sdk == null )
+        {
+            promise.resolve( targetingMaximumAdContentRatingToSet != null ? targetingMaximumAdContentRatingToSet : 0 );
+            return;
+        }
+
+        promise.resolve( sdk.getTargetingData().getMaximumAdContentRating().ordinal() );
```

Comment:
`sdk.getTargetingData().getMaximumAdContentRating()` can return `null` and thus cause NPE.